docs/learn/learn-governance.md

```diff
@@ -437,6 +483,62 @@ Fast-tracked referenda are the only type of referenda that can be active alongsi
 referendum. Thus, with fast-tracked referenda it is possible to have two active referendums at the
 same time. Voting on one does not prevent a user from voting on the other.
 
+In governance v2, a new successor committee was introduced, known as the "Polkadot Fellowship", to replace the Technical Committee. It will serve both the Polkadot and Kusama networks. See additional details below.
+
+## Polkadot Fellowship
+
+The Fellowship is a mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Polkadot network and protocol. This is accomplished by associating a rank with members to categorize the degree to which the system expects their opinion to be well-informed, of a sound technical basis and in line with the interests of Polkadot.
+
+Unlike the current Technical Collective it is designed to be far broader in membership (i.e. to work well with even tens of thousands of members) and with far lower in barriers to entry (both in terms of administrative process flow and expectations of expertise). Becoming a candidate member in the Fellowship is as easy as placing a small deposit.
+
+Members of the Fellowship can vote on any given Fellowship proposal and the aggregate opinion of the members (weighted by their rank) constitutes the Fellowship's considered opinion.
+
+The mechanism by which the Fellowship votes is the same (Substrate pallet) as what is used for Polkadot stakeholder voting for a proposed referendum.
+
+### Ranking System
+
+So how exactly does this ranking system work?
+
+To prevent a small group of participants from gaining effective control over the network this system will adhere to three main principles:
```

Comment:
```suggestion
To prevent a small group of participants from gaining effective control over the network, this system will adhere to three main principles:
```